### PR TITLE
DM-35441: pyproject, improved directory structure

### PR DESCRIPTION
**Pull request by pkubanek**
- pyproject translation, M1M3GUI module
- Moved files, create M1M3 and GUI/SAL folders

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,40 @@
-# Python GUI for M1M3 Support System
+# Python EUI/GUI for M1M3 Support System
+
+Engineering / User Graphical Interface for cRIO systems. Provides generics GUI
```
Comment (teweitsai) [inline README.md:3]: generics  --> generic

File: README.md [hunk 2]
```diff
@@ -1,14 +1,40 @@
...
+* __M1M3__ widgets for M1M3 support system
+* __M1M3TS__ widgets for M1M3 thermal system
+* __VMS__ graphs for [ts_VMS](https://github.com/lsst-ts/ts_VMS)
 
 ## Applications
 
 Applications are located in bin directory. Before running those, please make
 sure that required Python packages (in python directory) are available:
 
+```
```
Comment (teweitsai) [inline README.md:25]: You could add the syntax hint of language (see the section of "Syntax Highlighting"):
https://www.markdownguide.org/extended-syntax/

File: README.md [hunk 3]
```diff
@@ -1,14 +1,40 @@
...
 python3.8 -c "from lsst.ts.cRIOpy.GUI import *"
+```
 
 shall pass without error.
 
+
+```
```
Comment (teweitsai) [inline README.md:32]: Same above.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,40 @@
...
+pip install .
+```
+
+shall install those commands and packages needed for running them for you.
```
Comment (teweitsai) [inline README.md:36]: shall  --> Shall

File: README.md [hunk 5]
```diff
@@ -39,6 +65,10 @@ Python 3.8
 [asyncqt](https://pypi.org/project/asyncqt)
 [LSST ts_salobj](https://github.com/lsst-ts/ts_salobj)
 
+# SAL binding
+
+The GUI/EUI
```
Comment (teweitsai) [inline README.md:70]: This is an unfinished paragraph.

File: pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+dynamic = [ "version" ]
+dependencies = [ "pyside2", "asyncqt", "numpy", "astropy" ]
+
+[project.scripts]
+M1M3GUI = "lsst.ts.cRIOpy.M1M3GUI:run"
+M1M3TSGUI = "lsst.ts.cRIOpy.M1M3TSGUI:run"
+VMSGUI = "lsst.ts.cRIOpy.VMSGUI:run"
+VMSlogger = "lsst.ts.cRIOpy.VMSlogger:run"
+
+
```
Comment (teweitsai) [inline pyproject.toml:20]: You only need one empty line.

**Review by teweitsai — APPROVED** [review]
Looks good. I assume there should be no code change here because they look more like the code reorganization. Let me know if you want me to review a specific part of code. Thanks!

File: README.md (hunk 4 above)
Comment (pkubanek) [reply README.md:36]: Reworded

**Pull request merged**